codesydney/codesydney
# DEV-001 Express App

## Rationale

Scaffolding the backend project for Code.Sydney with Node, Express and Typescript. Built a server that initialise the app on port 5000.

## Advice for Reviewers & Testing Notes

- This project uses typescript, please ensure to install typescript and ts-node globally
```
npm install -g ts-node
npm install -g typescript
```
- Checkout the branch DEV-001-express-app
- Run ```yarn``` to initialise all the installed packages
- To start the server on port 5000 either run ```yarn run start``` or ```yarn run server```
- For development it is recommended to use ```yarn run server``` as it hot reloads.
- If everything is all good in the hood, you should see the the **App started on PORT 5000**

## Screenshots:

![image](https://user-images.githubusercontent.com/23388566/77821107-b6acab80-713b-11ea-8c07-1b53741222a7.png)

## Task Name

DEV-001 - Create a Backend server with Express


## Linting Checklist
The following is a list of linting rules that ESLint is not currently linting for. Please make sure your code conforms to this list. Examples/definitions of the rules can be found [here](https://hireup.atlassian.net/wiki/pages/viewpageattachments.action?pageId=618037301&metadataLink=true&preview=/618037301/618168365/PR_Checklist.pdf)
